src/database/seeders/TaskSeeder.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class TaskSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('tasks')->insert([
+            [
+                'start' => '2020-10-06 10:30:00',
```

Comment:
Could you change the days or times to have more variety for when we will make chars